node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/component/AdminComponent.java

```diff
@@ -13,6 +15,11 @@
      */
     void enable();
 
+    /**
+     * @return NodeAdminStateUpdater used by the REST API
```

Comment:
Why is this needed? NodeAdminStateUpdater is injected into REST API becaue NodeAdminProvider is Provider of NodeAdminStateUpdater?